Tests/Tests/Event Logging/EventLoggingTests.swift

```diff
@@ -40,7 +41,7 @@ class EventLoggingTests: XCTestCase {
     }
 
     func testThatAllFilesAreEventuallyUploaded() throws {
-        throw XCTSkip("This test seems to be flaky")
+        XCTExpectFailure("This test seems to be flaky")
```

Comment:
`XCTSkip` will always skip the test, but `XCTExpectFailure` will still try to run the test but expect it to fail. If the test succeeds, it will show as a failing test so that we're alerted that it's working correctly now and the `ExpectFailure` can be removed. 

https://www.avanderlee.com/swift/xctexpectfailure-expected-failures/